clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1100,6 +1100,12 @@ class raw2trace_t {
                           DR_PARAM_OUT bool *last_bb_handled,
                           DR_PARAM_OUT bool *flush_decode_cache);
 
+    /**
+     * Called for each record in an output buffer prior to writing out the buffer.
+     */
+    virtual void
```

Comment:
> possibly trim nearly-empty start-unscheduled threads.

Add to docx how the above can be done by sub-classes using this interface: whether `entry` can/should be modified etc.

Reply:
Done.